JSLinted

choyiny opened the pull request:
Still has errors, but don't know how to fix
Unexpected 'for'.
        for (i = 0; i < parameterArray.length; i += 1) {
line 164 column 8Unexpected 'for'.
        for (var i = 0; i < textNodes.snapshotLength; i++) {
line 164 column 13Unexpected 'var'.
        for (var i = 0; i < textNodes.snapshotLength; i++) {

JSLinted via JSLint.com

Commit:
JSLinted

Still has errors, but don't know how to fix
Unexpected 'for'.
        for (i = 0; i < parameterArray.length; i += 1) {
line 164 column 8Unexpected 'for'.
        for (var i = 0; i < textNodes.snapshotLength; i++) {
line 164 column 13Unexpected 'var'.
        for (var i = 0; i < textNodes.snapshotLength; i++) {

smcgregor commented:
Did you change all the white space? All the lines show as different, which makes it difficult to review.

Commit:
change spaces

sammyshj commented on javascripts/content_scripts/privly.js:
```diff
@@ -847,7 +837,7 @@ var privly = {
       "localhost:3000\\/" + //localhost:3000
       domainRegexp +
-      ")(\\S){3,}/[^\\s]*\\b","gi");
+      ")(\\S){3,}/[^\\s]*\\b", "gi");
```
The `"gi"` should be in the next line. This adds consistency to the fixes done previously at: https://github.com/hkminegod/privly-chrome/blob/master/javascripts/content_scripts/privly.js#L121

Commit:
add consistency to format

The "gi" should be in the next line. This adds consistency to the fixes done previously at: https://github.com/hkminegod/privly-chrome/blob/master/javascripts/content_scripts/privly.js#L121

smcgregor commented on javascripts/content_scripts/privly.js:
```diff
@@ -296,17 +295,16 @@ var privly = {
+          if (a.textContent.indexOf("privlyInject1") <= 0 || !privly.correctIndirection.testAndCopyOver(a, a.textContent)) {
             notUpdated.push(a);
           }
-      });
```
This seems like a strange change, since it is actually closing out the parenthesis from two spaces to the left.

smcgregor commented:
I just checked every line, and I only have the one question that I left inline. Can you please address the question then update the pull request so that it targets our `develop` branch instead of our master branch?

Regarding your question above, I'm not really sure what is causing that. I'll take a look if you update the PR to target the develop branch.
